vars.tf

```diff
@@ -137,3 +137,21 @@ variable "backup_retention_period" {
   description = "How long to store RDS DB backups"
   default     = null
 }
+
+variable "accessLog_enabled" {
+  type        = bool
+  description = "Enables Keycloak access log"
+  default     = false
```

Comment:
@jurakp I think we might want these values to default to `true` for the time being. @ianwestcott does that sound right to you?

Reply:
No problem ;)
With defaults turned to true:
PROS: access log is available all the time
CONS: some small impact on performance of course

With defaults turned of (status quo): neg ^^^

@ianwestcott What's your opinion. Maybe it is better to turn on by default.

Reply:
Thanks @jurakp I chatted with Ian and he makes a good point that we'll want to control this setting per environment. So we can leave the default as `false` and only enable logging for dev. Approving the PR as it stands!